GitCommands/RevisionReader.cs

```diff
@@ -19,18 +19,23 @@ namespace GitCommands
     [Flags]
     public enum RefFilterOptions
     {
-        None                    = 0x000,
-        Branches                = 0x001,    // --branches
-        Remotes                 = 0x002,    // --remotes
-        Tags                    = 0x004,    // --tags
-        Stashes                 = 0x008,    //
-        All                     = 0x00F,    // --all
-        Boundary                = 0x010,    // --boundary
-        ShowGitNotes            = 0x020,    // --not --glob=notes --not
-        NoMerges                = 0x040,    // --no-merges
-        FirstParent             = 0x080,    // --first-parent
-        SimplifyByDecoration    = 0x100,    // --simplify-by-decoration
-        Reflogs                 = 0x200,    // --reflog
+        None = 0x000,
```

Comment:
After removing the whitespace (-0), the `#pragma warning dis/enable SA1025` should be removed, too.